src/kudu/rpc/service_pool.cc

```diff
@@ -129,14 +129,18 @@ void ServicePool::RejectTooBusy(InboundCall* c) {
   KLOG_EVERY_N_SECS(WARNING, 300) << err_msg;
   c->RespondFailure(ErrorStatusPB::ERROR_SERVER_TOO_BUSY,
                     Status::ServiceUnavailable(err_msg));
-  DLOG(INFO) << err_msg << " Contents of service queue:\n"
-             << service_queue_.ToString();
+  VLOG(2) << err_msg << " Contents of service queue:\n"
```

Comment:
I think we can track, through a member variable in ServicePool, when a request is rejected. The contents of the queue can be dumped (at a regular log level) when a request is rejected the first time. The request rejected can then e unset the next time when a request gets accepted.

So, overall:
if (!request_rejection_logged_) {
  request_rejection_logged_ = true;
  LOG(INFO) << service_queue_.ToString();
}

Next time when a request is successfully queued in ServicePool::QueueInboundCall(...):
request_rejection_logged_ = false;